tests/settings.py

```diff
@@ -81,7 +81,8 @@
             'format': '[%(levelname)s] %(message)s'
         },
         'verbose': {
-            'format': '\n\n[%(levelname)s %(asctime)s] module: %(module)s, process: %(process)d, thread: %(thread)d\n%(message)s'
```

Comment:
same as previous, cancel changes to this file, and include this file to the excluded files of flake8 and isort